sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_items_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Newtab Items Daily
```

Comment:
Would it be feasible to derive this table from new_tab_visits?

Reply:
new_tab_visits currently does not have all the needed fields. We should eventually think of consolidating the  newtab etl processes. Perhaps as part of Nadia's revamp of the data models revisit all reporting requirements before we do it

Reply:
Makes sense, it looks like we mostly derive from stable tables so we'd need to make this columns available.